niceml/dlframeworks/tensorflow/metrics/objdetmetrics.py

```diff
@@ -35,10 +36,11 @@ class AvgNegPredObjDet:  # pylint: disable=too-few-public-methods
     """Negative Classification Values for object detection"""
 
     def __init__(self, name: str = "avg_neg_pred"):
-
+        """Constructor"""
```

Comment:
Please add more details. I think it's fine if you use the class docstring in combination with a description of the attributes.